.drone.star

```diff
@@ -1221,6 +1224,18 @@ def ocisService(type, tika_enabled = False, enforce_password_public_link = False
         environment["MICRO_REGISTRY_ADDRESS"] = "0.0.0.0:9233"
         environment["NATS_NATS_HOST"] = "0.0.0.0"
         environment["NATS_NATS_PORT"] = 9233
+    elif type == "keycloak":
+        environment["PROXY_AUTOPROVISION_ACCOUNTS"] = "true"
+        environment["PROXY_ROLE_ASSIGNMENT_DRIVER"] = "oidc"
+        environment["OCIS_OIDC_ISSUER"] = "https://keycloak:8443/realms/oCIS"
+        environment["PROXY_OIDC_REWRITE_WELLKNOWN"] = "true"
+        environment["WEB_OIDC_CLIENT_ID"] = "web"
+        environment["PROXY_USER_OIDC_CLAIM"] = "preferred_username"
+        environment["PROXY_USER_CS3_CLAIM"] = "username"
+        environment["OCIS_ADMIN_USER_ID"] = ""
+        environment["OCIS_EXCLUDE_RUN_SERVICES"] = "idp"
+        environment["GRAPH_ASSIGN_DEFAULT_USER_ROLE"] = "false"
+        environment["GRAPH_USERNAME_MATCH"] = "none"
     else:
         environment["WEB_UI_CONFIG_FILE"] = "%s" % dir["ocisConfig"]
```

Comment:
these envs should be for all setups. Also please, try to merge https://github.com/owncloud/web/pull/10530 first then forward port to master
```suggestion
    environment["WEB_UI_CONFIG_FILE"] = "%s" % dir["ocisConfig"]
```